remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/view/RemixerFragment.java

```diff
@@ -84,6 +87,19 @@ public void onClick(View view) {
   }
 
   /**
+   *
+   * @param manager
+   * @param tag
+   * @return whether the fragment was shown or not.
+   */
+
+  @Override
+  public void show(FragmentManager manager, String tag) {
```

Comment:
Instead of overriding this, I would wrap it into a showRemixer method, don't know if the underlying classes end up calling show directly at some point and we may be causing another bug. Seems safer.